ext/manifest.json

```diff
@@ -42,7 +42,8 @@
         "<all_urls>",
         "storage",
         "clipboardWrite",
-        "unlimitedStorage"
+        "unlimitedStorage",
+        "nativeMessaging"
```

Comment:
May be another case for using optional_permissions for nativeMessaging, since it's a non-essential functionality.

Reply:
Unfortunately this permission doesn't work in optional_permissions, at least on Firefox.